applications/ContactStructuralMechanicsApplication/custom_utilities/contact_utilities.h

```diff
@@ -226,7 +220,7 @@ class ContactUtilities
         {
             auto it_node = nodes_array.begin() + i;
             it_node->SetValue(NODAL_AREA, 0.0);
-            it_node->SetValue(NORMAL, zero_vect);
+            it_node->FastGetSolutionStepValue(NORMAL) = zero_vect;
```

Comment:
use ZeroVector instead. It is an expression!!

Reply:
Ok